_stbt/core.py

```diff
@@ -1630,8 +1653,9 @@ def draw(self, obj, duration_secs):
                 self.text_annotations.append(
                     {"text": obj, "duration": duration_secs * Gst.SECOND})
             elif isinstance(obj, MatchResult):
-                if obj.timestamp is not None:
```

Comment:
I imagine this check for `is not None` was so that the very first frame (with timestamp 0?) wouldn't be excluded. It's unlikely to matter though.